src/metabase/db.clj

```diff
@@ -53,8 +53,10 @@
 
 (defn setup-db!
   "Do general preparation of database by validating that we can connect. Caller can specify if we should run any pending
-  database migrations. If DB is already set up, this function will no-op. Thread-safe."
-  []
+  database migrations. If DB is already set up, this function will no-op. Thread-safe.
+  Callers must decide whether or not to create sample content during migrations."
```

Comment:
The value is arguably minimal but I wanted it to be an explicit decision for every caller because in many cases where it's false, it's because we're skipping it to make tests faster. I want to highlight the danger in those cases. And I don't want to make it true by default because there are cases where that is obviously wrong e.g. load-from-h2 or importing serialized content.

Reply:
I've come around, I like every site having an explicit choice coupled with an explicit explanation comment.